app/controllers/panel_controller.rb

```diff
@@ -4,8 +4,8 @@ class PanelController < ApplicationController
   include DocumentsHelper
 
   before_action :authenticate_user!
-  before_action -> { redirect_to_root_unless_user(:staff_or_any_delegate?) }
```

Comment:
Not happy if global permissions are removed.

Reply:
Okay, added back, but using `can_access_panel?` method now as that is more appropriate.